markdown_lint/__init__.py

```
@@ -0,0 +1,8 @@
+"""Markdown linter and fixer with support for common style issues."""
+
+from dataclasses import dataclass, field
```

Ruff (0.12.2)

`dataclasses.dataclass` imported but unused

(F401)